storage/mysql/provider.go

```diff
@@ -107,3 +120,32 @@ func (s *mysqlProvider) AdminStorage() storage.AdminStorage {
 func (s *mysqlProvider) Close() error {
 	return s.db.Close()
 }
+
+func registerTLSConfig() error {
```

Comment:
Maybe rename this to `registerMySQLTLSConfig` to be clear about what it's doing & why.